### PR TITLE
Add table size check to quickstart integration test [VS-501]

### PR DESCRIPTION
**Pull request by rsasch**
Successful run here: https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/96ceaff0-810a-4e79-a59d-2f5d8405d487

### CODE REVIEW

File: scripts/variantstore/wdl/GvsQuickstartIntegration.wdl [hunk 1]
```diff
@@ -89,6 +89,14 @@ workflow GvsQuickstartIntegration {
             expected_output_csv = expected_output_prefix + "cost_observability_expected.csv"
     }
 
+    call AssertTableSizesAreExpected {
+        input:
+            go = GvsUnified.done,
+            dataset_name = BuildGATKJarAndCreateDataset.dataset_name,
+            project_id = project_id,
+            expected_output_csv = "gs://gvs-internal-quickstart/integration/2022-07-05/table_sizes_expected.csv"
```
Comment (gbggrant) [inline scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:97]: This should be 
```suggestion
            expected_output_csv = expected_output_prefix + "table_sizes_expected.csv"
```

**Review by gbggrant — APPROVED** [review]
One suggestion.

File: scripts/variantstore/wdl/GvsQuickstartIntegration.wdl (hunk 1 above)
Comment (rsasch) [reply scripts/variantstore/wdl/GvsQuickstartIntegration.wdl]: done

File: scripts/variantstore/wdl/GvsQuickstartIntegration.wdl [hunk 2]
```diff
@@ -283,6 +291,56 @@ task AssertCostIsTrackedAndExpected {
...
+    }
+
+    command <<<
+        set -o errexit
+        set -o nounset
+        set -o pipefail
+        set -o xtrace
+
+        echo "project_id = ~{project_id}" > ~/.bigqueryrc
+        bq query --location=US --project_id=~{project_id} --format=csv --use_legacy_sql=false \
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:318]: TOL I think if you start the double quote on this line you could get rid of all the trailing backslashes. Maybe.

**Review by mcovarr — APPROVED** [review]

**Review by RoriCremer — APPROVED** [review]

**Pull request merged**